tests/unit/test_utils.py

```diff
@@ -8,6 +8,7 @@
 from django.http import HttpRequest
 from django.template import Context
 from django.test.utils import override_settings
+from utils import TestCase
```

Comment:
All these `utils` imports are really "local" imports. I hope we won't get bitten by them, one day. It may make sense to convert all them to relative imports.

Reply:
This sounds like a separate PR to me?

Reply:
Yeah, this one does. :100: